### ISSUE TITLE
"White noise" screen was broken by recent changes

### ISSUE THREAD

**Issue by semi-simple** [MiSTer-devel/Menu_MiSTer#31]
The "white noise" screen has been completely broken by the latest releases.

- Latest menu.rbf (SHA256: 2e7c38b873289ab2f766122faf5623a8e3c33850ae444890ca7558e791598fc5): white noise "blinks", has random black chunks in the screen etc (sometimes doesn't show up at all).

This problem happens both on the VGA out and on HDMI.

- menu_20230223.rbf (SHA256: 33ef21a1db4e191d8669a3579ac6883fd4e3f9d0b6753708122210b6d611dddf): white noise works fine on VGA and HDMI

**Comment by sorgelig** [comment]
Noise was always like that from build to build because Quartus tries to optimize intended cells loop.
I test (usually) the noise before release menu.rbf. If you try to compile yourself then result may be different.

**Comment by semi-simple** [comment]
What is strange is that that exact same build seems to work for others. For me it's completely broken. Is there anything  (maybe config related) you can think of that  could make it work for others? I confirm that if I just use the old menu_20230223.rbf everything is ok.

**Comment by sorgelig** [comment]
The best way would be to prevent optimization of that chain, but i didn't find the way to do it.

**Comment by bootsector** [comment]
Having this issue with the latest menu:

https://cdn.discordapp.com/attachments/806759728885006356/1301860736691339285/IMG_2560.mov?ex=672603cb&is=6724b24b&hm=d44560cc248bfadb858672e9aff05cda4f653435f9abf6843d0dd5b452c7b8f8&

**Comment by sorgelig** [comment]
You can't disable noise mode for CRT, but you can use vga_scaler option in INI to output HDMI picture over VGA (if your CRT supports such resolution).

**Comment by TheJesusFish** [comment]
That build has the new framework so it’s not ready for prime time. However, it does show that the issue may be a metastability thing.

**Comment by sorgelig** [comment]
It's not really metastability issue. It's because RNG uses a trick of circular LE chain which Quartus tries to optimize.

**Comment by rittwage** [comment]
This happened again with the recent release...

**Comment by TheJesusFish** [comment]
Yeah, and it seems to be for everyone, not silicon lottery. Might just need to run another build with a different seed.

**Comment by sorgelig** [comment]
or need to find the way to make RNG preventing optimization from Quartus.

**Comment by sorgelig** [comment]
try this build (nothing new, just another build):
[menu.zip](https://github.com/user-attachments/files/25619319/menu.zip)

**Comment by bootsector** [comment]
> try this build (nothing new, just another build): [menu.zip](https://github.com/user-attachments/files/25619319/menu.zip)

This is working great on all of my MiSTers!
